kubernetes/kylin-job-statefulset.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  annotations: {}
+  name: kylin-job
+  namespace: kylin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kylin
+      type: job
+  serviceName: kylin
+  template:
+    metadata:
+      labels:
+        app: kylin
+        type: job
+    spec:
+      containers:
+        - image: 'apachekylin/apache-kylin:3.0.0-cdh57'
+          imagePullPolicy: Always
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - bash
+                  - '-c'
+                  - |
+                    set -ex
+                    # initialize the keytab
+                    kinit -kt /home/kylin/kylin.keytab kylin
```

Comment:
Maybe we can create a bootstrap.sh to includes these command, and exec the shell script in here.